plugins/integrations/kubernetes-service/src/main/java/com/cloud/kubernetes/cluster/utils/KubernetesClusterUtil.java

```diff
@@ -218,7 +221,10 @@ public static boolean isKubernetesClusterServerRunning(final KubernetesCluster k
         boolean k8sApiServerSetup = false;
         while (System.currentTimeMillis() < timeoutTime) {
             try {
-                String versionOutput = IOUtils.toString(new URL(String.format("https://%s:%d/version", ipAddress, port)), StringUtils.getPreferredCharset());
+                URL url = new URL(String.format("https://%s:%d/version", ipAddress, port));
+                HttpsURLConnection con = (HttpsURLConnection)url.openConnection();
```

Comment:
should we configure it to ignore ssl cert validation?

Reply:
and get more static analysis reports as issues?